### PR TITLE
Jakz/custom notification

### PR DESCRIPTION
**Pull request by jakzaizzat**
### Summary of Changes

Provide a brief description of the changes made in this PR. Explain the problem you are fixing or the feature you are adding.

### Demo or Before/After Pics

- If this is a new feature, include screenshots or recordings of the feature in action.
- If this PR makes visual changes, include before and after screenshots.

| Before                                     | After                                      |
| ------------------------------------------ | ------------------------------------------ |
|  |  |

### Edge Cases

List any common edge cases that you have considered and tested.

### Testing Steps

Provide steps on how the reviewer can test the changes.

### Checklist

Please make sure to review and check all of the following:

- [ ] I've tested the changes and all tests pass.
- [ ] (if web) I've tested the changes on various desktop screen sizes to ensure responsiveness.
- [ ] (if mobile) I've tested the changes on both light and dark modes.

### CODE REVIEW

File: apps/mobile/src/components/Notification/Notifications/GalleryAnnouncement.tsx [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+      }
+    `,
+    queryRef
+  );
+
+  const { title, description, ctaText, ctaLink, imageUrl } = notification;
+
+  const handlePress = useCallback(() => {
+    if (!ctaLink) return;
+    Linking.openURL(ctaLink);
```
Comment (Robinnnnn) [inline apps/mobile/src/components/Notification/Notifications/GalleryAnnouncement.tsx:49]: only small thing is `ctaLink` might be a relative link, like `/jakz`, in which case we'd want to deep link to their profile, with a slightly modified version of `handleDeepLinkNavigation`

for now though we can assume `ctaLink` will always be an external link

**Review by Robinnnnn — APPROVED** [review]

File: apps/mobile/src/components/Notification/Notifications/GalleryAnnouncement.tsx (hunk 1 above)
Comment (jakzaizzat) [reply apps/mobile/src/components/Notification/Notifications/GalleryAnnouncement.tsx:49]: Sounds good! Will add deep link support in future. I'll create the ticket to keep track this

Comment (Robinnnnn) [reply apps/mobile/src/components/Notification/Notifications/GalleryAnnouncement.tsx:49]: legend
